src/OmnichannelChatSDK.ts

```diff
@@ -422,14 +423,18 @@ class OmnichannelChatSDK {
                     response: "OCClientSessionInitFailed"
                 };
 
+                if ((error as any)?.isAxiosError && (error as any).response?.headers?.errorcode.toString() === OmnichannelErrorCodes.WidgetUseOutsideOperatingHour.toString()) {
+                    exceptionDetails.response = OmnichannelErrorCodes[OmnichannelErrorCodes.WidgetUseOutsideOperatingHour].toString();
```

Comment:
Can we add a exceptionDetails.message="Widget used outside of operating hours" to better explain?